content/en/blog/recap-2022/index.md

```diff
@@ -0,0 +1,137 @@
+
+- As the home of such important assets, GitHub repositories should be at the top of your list of security priorities. However, many people fail to put in place even basic measures to protect source code repositories. [Read abou](https://falco.org/blog/falco-plugin-github/)t three risks to GitHub repos and how you can reliably detect them, as they happen, using Falco open source security. 
+
+- Before 0.32.1, Falco could not work with gVisor monitored sandboxes because it was not possible to install a kernel module or eBPF probe in such an environment. But wouldn't it be great to leverage the stream of system call information that gVisor collects through its powerful monitoring system directly in Falco? This is exactly what became possible with gVisor release 20220704.0 and Falco 0.32.1. [Learn how to integrate gVisor and Falco on Docker](https://falco.org/blog/intro-gvisor-falco/). 
+
+- With Falco 0.32.1, you can now run Falco on Apple ARM M1 CPUs. It requires a Linux virtual machine (VM) since Falco doesn't run on OSX, but it is pretty straightforward. Here are the step by step [instructions.](https://falco.org/blog/falco-apple-silicon/) 
+
+- Different technologies are used on a daily basis, and tools like Vagrant, Terraform, Ansible, plus many more allow us to create and destroy digital resources in a matter of minutes, if not seconds. However, if you keep changing your running environment, you might need to calibrate your workloads to these new changes. This is especially true when you deploy operating system-dependent applications. In other words, every time you deploy an application like Falco there's a chance that you need to compile a new module or eBPF probe to get along with the current underlying kernel. This is the first of a series of posts introducing some interesting techniques using Falco to generate the much needed driver and how you can make it available for your deployments. [Learn how to build your own Falco Drivers for Debian.](https://falco.org/blog/falco-driverkit-debian-docker/) 
+
+- Giant Swarm simplifies the maintenance of the software stack within Kubernetes clusters by using its App Platform technology. Leverage this to easily deploy Falco, either individually or as part of Giant Swarm's Security Pack, to secure a managed Kubernetes service. [Learn how to deploy and manage applications across hybrid environments.](https://falco.org/blog/giantswarm-app-platform-falco/)
+
+- At the beginning of 2022, Falco introduced a game changing feature, Falco Plugins. They allow Falco to monitor and trigger alerts for any kind of event. Since the launch of the new plugin framework the Falco community has collaborated to create plugins for GitHub, AWS CloudTrail and Okta. 
```

Comment:
From a logical perspective, I would mention the plugin system before the GitHub security bullet point above, since that is implemented through the GitHub plugin.

Reply:
updated!